app.py

```diff
@@ -8,27 +8,50 @@
 from flask import Flask, flash, request, redirect, render_template, url_for, send_from_directory
 from werkzeug.utils import secure_filename
 from converters.json2glm import json2glm
+from converters.json2png import json2png
 
 app = Flask(__name__)
 app.config.from_object("settings.config.DevelopmentConfig")
 
+
 def allowed_file(filename):
     name, extension = os.path.splitext(filename)
     extension = extension[1:].lower()
     return extension in app.config['ALLOWED_EXTENSIONS']
 
+
 @app.route('/')
 def index():
     return render_template('home.html')
 
+
 @app.route('/convert')
 def upload_form():
     return render_template('convert.html')
 
+
 @app.route('/', methods=['POST'])
 def upload_file():
     convert_from = request.form["convertFrom"]
     convert_to = request.form["convertTo"]
+# json2png default parameters
+    output_type = 'summary'
+    with_nodes = False
+    resolution = "300"
+    size = "300x200"
+    limit = None
+    if (convert_from == "json" and convert_to == "png"):
+        # json2png custom options
+        if (request.form["size"] != ""):
```

Comment:
Instead of setting the defaults above and then doing all these if statements, use the `.get` method of the dict and supply a default.
i.e.:
```python
size = request.form.get('size', '300x200')
```

Reply:
The dictionary `.get()` method seems to set a default value if the key doesn't exist rather than checking if the value for the corresponding key exists.

Reply:
I generated a defaults dict for every file that takes more than just the file_path parameter (file_in) and for now we only have json2png that fits this description. Then if the value from the request is an empty string, it is replaced by the default value. I also pass a dictionary instead of individual arguments to each function.

Reply:
> The dictionary `.get()` method seems to set a default value if the key doesn't exist rather than checking if the value for the corresponding key exists.

But that's exactly the point. It completely bypasses the need to check the value. If the key doesn't exist, it sets the default value, which should be coming from the defaults dict.

Don't take these suggestions as the literal approach.

If it was literal it'd be something like: 
```python
size = request.form.get('size', defaults['size'])
```